### PR TITLE
fix(typo): fix some typos

### PR DESCRIPTION
**Pull request by gitforhlp**
fix word error

### CODE REVIEW

File: en/api/action.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ dispatchAction({
 Downplay specified data graphics.
 
 ```ts
-// If downplay series:
+// If downplay series（in the `sunburst` series,`downplay` has been deprecated since `v5.0.0`, please use `sunburstUnhighlight` instead）：
```
Comment (plainheart) [inline en/api/action.md:69]: Please revert this change.

File: en/api/action.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Chart actions supported by ECharts are triggered through [dispatchAction](~echar
 Highlights specified data graphics.
 
 ```ts
-// If highlight series:
+// If highlight series（in the `sunburst` series,`highlight` has been deprecated since `v5.0.0`, please use `sunburstHighlight` instead）：
```
Comment (plainheart) [inline en/api/action.md:44]: Please revert this change.

**Review by plainheart — CHANGES_REQUESTED** [review]

File: en/api/action.md (hunk 2 above)
Comment (gitforhlp) [reply en/api/action.md]: I have  reverted it , please review again

File: en/api/action.md (hunk 1 above)
Comment (plainheart) [reply en/api/action.md:69]: This issue is not solved yet.

File: zh/api/action.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ ECharts 中支持的图表行为，通过 [dispatchAction](~echartsInstance.disp
 高亮指定的数据图形。
 
 ```ts
-// 如果要高亮系列：
+// 如果要高亮系列（旭日图`sunburst`,从 `v5.0.0` 开始`highlight` 已被弃用，请使用 `sunburstHighlight` 代替）：
```
Comment (plainheart) [inline zh/api/action.md:43]: Should remove this change as well.

File: zh/api/action.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ dispatchAction({
 取消高亮指定的数据图形。
 
 ```ts
-// 如果要取消高亮系列：
+// 如果要取消高亮系列（旭日图`sunburst`,从 `v5.0.0` 开始`downplay` 已被弃用，请使用 `sunburstUnhighlight` 代替）：
```
Comment (plainheart) [inline zh/api/action.md:67]: Should remove this change as well.

**Review by plainheart — CHANGES_REQUESTED** [review]

File: en/api/action.md [hunk 5]
```diff
@@ -294,7 +294,7 @@ dispatchAction({
 
 ### takeGlobalCursor(Action)
 
-Activate or inactivate `dataZoom` buttom in `toolbox`.
+Activate or inactivate `dataZoom` bottom in `toolbox`.
```
Comment (plainheart) [inline en/api/action.md:297]: Should be `button`.

**Review by plainheart — CHANGES_REQUESTED** [review]

**Review by plainheart — APPROVED** [review]
